### ISSUE TITLE
AppIntro2 - Next/Done Button is clipped

### ISSUE THREAD

**Issue by Juchar** [AppIntro/AppIntro#354]
**AppIntro Version**: 4.1.0

**Device/Android Version**: fails on LG G5 6.0.1

**Issue details / Repro steps / Use case background**: Simply create an activity with a slide and press the button to proceed to the next slide or finish the intro.

**Your Code**: 
```
public class IntroActivity extends AppIntro2 {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        addSlide(AppIntroFragment.newInstance("Title", "Description", R.drawable.my_drawable,
                Color.parseColor("#3F51B5")));
    }
}
```

**Stack trace / LogCat**: Not applicable

![screenshot_2016-11-15-18-05-39](https://cloud.githubusercontent.com/assets/14292982/20315574/ba268ac0-ab5e-11e6-908f-565f761225eb.png)

**Comment by Andrew-Quebe** [comment]
This was originally noticed by @HeinrichReimer, but I closed the issue knowing it existed...but it got forgotten about due to the issue being closed. My mistake. It will be fixed in the next release...thanks for bringing it up again.

**Comment by shietoo** [comment]
**AppIntro Version: 4.2.0

Device/Android Version: fails on Samsung Note 5  6.0.1**

        addSlide(GuideFragment1.newInstance(R.drawable.a1));

I use a fragment to set background,the issue still exist!
    

![screenshot_20170608-170455](https://user-images.githubusercontent.com/18546799/26921068-79231d80-4c6d-11e7-975e-7d56cbffd0dd.jpg)

**Comment by paolorotolo** [comment]
@Juchar I can't reproduce the issue on Android 8, can you confirm it's still there?

**Comment by Anutha02** [comment]
@paolorotolo This issue should be fixed by #647 if it was still there
